html-aam/roles.html

```diff
@@ -12,30 +12,23 @@
 <body>
 
 
```

Comment:
Adding some notes to ease reviewers' understanding of the PR.

All of the tests that were removed from this existing file are now in the new files above.